### PR TITLE
Update Python library

### PR DESCRIPTION
**Pull request by zhenghuiwang**
Fixes three issues related to python library:

1. The `kfmd.openapi_client` pkg not found in published `kfmd` package. We have to refer the pacakge as `kfmd.openapi_client` instead of `openapi_client` in our source code. (Changes the `make` target for this)

2. For optional properties with `None` value, we need to `del` the key. Otherwise, the server returns 400.

3. Fixes the demo notebook and adds a test for the notebook.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/metadata/99)

### CODE REVIEW

File: sdk/python/tests/test.py [hunk 1]
```diff
@@ -107,6 +107,27 @@ def test_log_invalid_artifacts_should_fail(self):
     ))
     self.assertRaises(ValueError, e.log_output, artifact2)
 
+  def test_log_metadata_successfully_with_minimum_inforamtion(self):
+    ws1 = metadata.Workspace(backend_url_prefix="127.0.0.1:8080", name="ws_1")
+
+    r = metadata.Run(workspace=ws1, name="first run")
+
+    e = metadata.Execution(name="test execution", workspace=ws1, run=r)
+    assert e.id
```
Comment (neuromage) [inline sdk/python/tests/test.py]: Are we checking that this is not none or true? If so, prefer `self.assertIsNotNone(e.id)` or similar.

Same below

File: sdk/python/kfmd/metadata.py [hunk 2]
```diff
@@ -14,8 +14,8 @@
 
 import datetime
 import json
-import openapi_client
-from openapi_client import Configuration, ApiClient, MetadataServiceApi
+import kfmd.openapi_client as openapi_client
+from kfmd.openapi_client import Configuration, ApiClient, MetadataServiceApi
```
Comment (neuromage) [inline sdk/python/kfmd/metadata.py]: style nit: prefer to import modules: 
http://google.github.io/styleguide/pyguide.html#22-imports

File: sdk/python/tests/test.py (hunk 1 above)
Comment (zhenghuiwang) [reply sdk/python/tests/test.py]: Done!

File: sdk/python/kfmd/metadata.py (hunk 2 above)
Comment (zhenghuiwang) [reply sdk/python/kfmd/metadata.py]: Done.

**Comment by neuromage** [comment]
/lgtm
/approve
Thanks @zhenghuiwang !

**Pull request merged**